SPECS/metal3-cluster-api-provider-metal3/README.md

```diff
@@ -0,0 +1,18 @@
+Adds the three packages for metal3 containers who components are written in
+golang:
+
+- metal3/ip-address-manager v0.0.4
+- metal3/cluster-api-controller-metal3 v0.4.0
+- metal3/baremetal-operator v0.4.0
+
+Build steps are as close to the original Dockerfiles as possible. Two major
+differences:
+
+1. Instead of pinning the golang tools to a specific version, >= requirement is
+used. v1.14 for BMO was specifically "not found" and there shouldn't be any harm
+in using a higher backwards-compatible version.
```

Comment:
Might be better to pin to a higher version that we know works to avoid future build fails.